liquidctl/cli.py

```diff
@@ -117,6 +118,7 @@
     '--legacy-690lc': bool,
     '--non-volatile': bool,
     '--direct-access': bool,
+    '--fan-mode': str.lower,
```

Comment:
Directly parse it here with `fan_mode_parser` (similarly to `--alert-color`).

Reply:
fixed in 522976b